dashboard/app/views/layouts/application.html.haml

```diff
@@ -32,7 +32,7 @@
       = tag 'meta', property: 'og:title', content: title
       = tag 'meta', property: 'og:image', content: CDO.studio_url(CDO.shared_image_url('share/girl_cropped.jpg'), 'https:')
       = tag 'meta', property: 'og:description', content: t('share.description')
-  %body
+  %body{class: @body_classes}
```

Comment:
i'm guessing HAML will gracefully handle the case where `@body_classes` is nil, but just double-checking in case we should add a fallback or something similar here 😄

Reply:
Good question.  I confirmed that a non-Oceans level has no additional classes added to `body`.